main.go

```diff
@@ -195,4 +195,5 @@ func mapCRDAvailability(crdNames ...string) map[string]bool {
 // This function is a placeholder for offering plugin integration
 func pluginAddToScheme(scheme *runtime.Scheme) {
 	controllers.DFAddToScheme(scheme)
+	controllers.DFCAddToScheme(scheme)
```

Comment:
- maybe a TODO comment here will be good stating atm we can't run the offering reconciler standalone?
- similar one at setupwatches as well

Reply:
We already have a comment above stating the function is a `placeholder`.

Similar to setupWatches